framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1609,6 +1613,7 @@ protected AnnotatedDeclaredType getImplicitReceiverType(ExpressionTree tree) {
                 if (path == null) {
                     // The path is null if the field is in a compilation unit we haven't
                     // processed yet. TODO: is there a better way?
+                    // This only arises in the Nullness Checker when substituting rawness.
```

Comment:
Why did you add this comment? What do you mean with `rawness`?

Reply:
That's odd... It's not added by me, I will check whether there is any merge issues on my branch.